### PR TITLE
Allow auctionResultsConnection to be filtered by new date fields

### PR DESCRIPTION
**Pull request by jpotts244**
Add the ability to filter the auctionResultsConnection by created date ranges 
Jira Ticket: https://artsyproduct.atlassian.net/browse/PURCHASE-1813

Ex of filtering queries:

`earliestCreatedYear: 2000, latestCreatedYear: 2001`
Screen Shot 2020-03-09 at 2 26 25 PM

`earliestCreatedYear: 1986, latestCreatedYear: 2001`
Screen Shot 2020-03-09 at 2 26 04 PM

### CODE REVIEW

File: _schema.graphql [hunk 1]
```diff
@@ -1,19 +1,19 @@
 directive @principalField on FIELD
 
-# Autogenerated input type of AddAssetToConsignmentSubmission
 input AddAssetToConsignmentSubmissionInput {
-  # A unique identifier for the client performing the mutation.
+  # The type of the asset
```
Comment (jpotts244) [inline _schema.graphql]: That pre-commit hook that runs generated a really large diff to the two schemas, is this expected? If not, any suggestions on getting around that? I was a little surprised to see such a large diff

Comment (sepans) [reply _schema.graphql]: I am guessing `ENABLE_CONSIGNMENTS_STITCHING` env variable is recently set to true on staging but you don't have it in your `.env` file so all the [convection](https://github.com/artsy/convection) stitched schema fields that are already added to master is being removed by `dump-schema`

Comment (jpotts244) [reply _schema.graphql]: Ah yes, that was exactly it. Thanks for the context!!

Comment (damassi) [reply _schema.graphql]: @jpotts244 - mind updating `.env.example` with this value?

Comment (jpotts244) [reply _schema.graphql]: will do ^

**Comment by zephraph** [comment]
This is so good 👏

**Pull request merged**